hera_cal/abscal_funcs.py

```diff
@@ -506,7 +506,7 @@ def delay_slope_lincal(model, data, antpos, wgts=None, df=9.765625e4, medfilt=Tr
     -------
     fit : dictionary containing delay slope (T_x) for each pol [seconds / meter].
     """
-    echo("...configuring linsolve data for delay_lincal", verbose=verbose)
+    echo("...configuring linsolve data for delay_slope_lincal", verbose=verbose)
```

Comment:
thanks for the fix.. i've been meaning to fix this specifically for months... haha

Reply:
Haha, no problem.

Code reviews are one thing, but I definitely understand abscal a lot better now that I've had to *modify* it.